doc/releases/migration-guide-4.0.rst

```diff
@@ -99,6 +99,11 @@ Other Subsystems
 Flash map
 =========
 
+ * ``CONFIG_SPI_NOR_IDLE_IN_DPD`` has been removed from the :kconfig:option:`CONFIG_SPI_NOR`
+   driver. An enhanced version of this functionality can be obtained by enabling
+   :ref:`pm-device-runtime` on the device (Tunable with
+   :kconfig:option:`CONFIG_SPI_NOR_ACTIVE_DWELL_MS`).
```

Comment:
nit: this entry should be added after the runtime PM support and the mentioned Kconfig option are actually added, so in the second commit, not the first one.